go/private/tools.bzl

```diff
@@ -0,0 +1,69 @@
+# Copyright 2014 The Bazel Authors. All rights reserved.
```

Comment:
s/2014/2017/

Will `go_fmt_test` and `go_fix_test` also be in this file in the future? tools.bzl is an unfortunately generic name, even though it is accurate. Maybe go_tools.bzl or tool_tests.bzl would be better.